src/go/k8s/pkg/resources/configmap.go

```diff
@@ -188,15 +188,16 @@ func (r *ConfigMapResource) createConfiguration(
 		}
 	}
 	if r.pandaCluster.Spec.Configuration.TLS.AdminAPI.Enabled {
-		cr.AdminApiTLS = config.ServerTLS{
+		adminTLS := config.ServerTLS{
```

Comment:
the TLS config has to have a name

Reply:
In this PR, let's leave the names empty. There is only one Admin listener. #1134 uses names for both listeners.